storage/include/storage_al.h

```diff
@@ -19,7 +19,11 @@
 #include <stddef.h>
 
 // platform HMAC and blob size
-#define PLATFORM_HMAC_SIZE BUFF_SIZE_32_BYTES
+#if defined(DEVICE_TPM20_ENABLED) && defined(ECDSA384_DA)
```

Comment:
With TPM 1.2 is the HMAC size always 32 bytes ?

Reply:
yes and currently we are only supporting TPM 2.0